back/app/services/navbar/populate_default_items_service.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+module Navbar
+  class PopulateDefaultItemsService
+    def call
+      ActiveRecord::Base.transaction do
+        create_default_navbar_items_with_pages
+        add_navbar_items_to_the_existing_pages
+      end
+    end
+
+    private
+
+    def create_default_navbar_items_with_pages
+      create_page_and_item(
+        type: "home",
+        ordering: 0,
+        page_title: { en: "Home" },
+        item_title: { en: "Home" },
```

Comment:
For these, you'll have to get the values from the translations. There's a `i18n_to_multiloc`method on the MultilocService to help you convert the translations to multilocs.